modules/rosa-sdpolicy-platform.adoc

```diff
@@ -5,6 +5,7 @@
 
 [id="rosa-sdpolicy-platform_{context}"]
 = Platform
+:productwinc: Red Hat OpenShift support for Windows Containers
```

Comment:
I assume this was for testing and isn't actually supposed to be here?

Reply:
No, I cannot get `include::_attributes/common-attributes.adoc[]` to work, so I went with a file-level attribute. Not important enough at this time to figure out what is going on.